pkg/agent/secondarynetwork/podwatch/podwatch_controller_init.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Antrea Authors
```

Comment:
@jianjuns separate file for init was introduced based on secondary network infra (OVS) init needs. I will revisit on this file needs while pulling in the OVS bridge init changes in.

Reply:
Then it should not be called "podwatch_controller_init.go", but maybe just "init.go" under "/secondarynetwork"? For now, I still suggest to move the code to "controller.go".